src/AppInstallerCLICore/Commands/RootCommand.cpp

```diff
@@ -173,6 +173,8 @@ namespace AppInstaller::CLI
             info << std::endl <<
                 "Windows: "_liv << Runtime::GetOSVersion() << std::endl;
 
+            info << Resource::String::Architecture << ": "_liv << Utility::ToString(Utility::GetSystemArchitecture()) << std::endl;

```

Comment:
We haven't enforced the output rule yet, but I think the Utility::ToString(Utility::GetSystemArchitecture()) needs to be wrapped inside LocIndString

Reply:
`Utility::ToString(Architecture)` could probably be changed to return a loc independent view.  If it turns out we need to localize it in the future, that can be its own problem.